woocommerce/woocommerce
# Fix a minor bug in code analyzer and improve logging for when no changes exist

### All Submissions:






### Changes proposed in this Pull Request:

There was a small bug in code-analyzer where it appears to hang and never finish, but actually it was just missing a call to `Logger.endTask()`. This makes you think the schema check never finishes if there is no actual change to output.

In fact in general its confusing to know if there were any changes, so I also added some logging for the empty states, just to make it clear that the task finished and found no changes.





### How to test the changes in this Pull Request:




1. Run a command such as `pnpm run analyzer -- lint release/7.5 "7.5.0" -b release/7.4`
2. See that there is never a infinite progress indicator and that the command finishes
3. See that for empty states such as templates/hooks/db that it logs some output stating no changes were found.
 


### Other information:




### FOR PR REVIEWER ONLY:
